src/vl-rich-data-table.js

```diff
@@ -19,6 +19,7 @@ import {VlRichDataSorter} from "./vl-rich-data-sorter.js";
  * @property {boolean} data-vl-multisort - Laat de gebruiker sorteren op meer dan 1 kolom.
  *
  * @slot filter - slot dat de velden bevat waarop gefilterd wordt. De formData van de search filter worden via een change event doorgegeven bij een wijziging.
+ * @property {boolean} data-vl-filter-closable - Attribuut dat de filter sluitbaar maakt en een knop getoond wordt om de filter te tonen en terug te verbergen.
```

Comment:
wanneer zou je eigenlijk ooit NIET de sluiten knop willen hebben? hoe krijg je het dan gesloten?

Reply:
ah met de filter knop toggelen dan .. cava

Reply:
Zonder het attribuut blijven beiden verborgen. Maar misschien kan @coemans nog wat toelichten wat het default gedrag best is en of dit attribuut misschien kan weggelaten worden. Wat ook wel zou kunnen is het gewoon afhankelijk maken van of je het filter slot hebt gebruikt of niet...

Reply:
Ok, comment hieronder brengt meer duidelijkheid.